ow_sim_tests/scripts/test_task_scoop_linear.py

```diff
@@ -0,0 +1,140 @@
+#!/usr/bin/env python
+
+# The Notices and Disclaimers for Ocean Worlds Autonomy Testbed for Exploration
+# Research and Simulation can be found in README.md in the root directory of
+# this repository.
+
+import rospy
+import roslib
+import unittest
+import owl_msgs.msg
+
+from action_testing import *
+from ow_lander import constants
+
+PKG = 'ow_sim_tests'
+TEST_NAME = 'task_scoop_linear'
+roslib.load_manifest(PKG)
+
+class TestTaskScoopLinear(unittest.TestCase):
+
+  @classmethod
+  def setUpClass(cls):
+    rospy.init_node("test_task_scoop_linear")
+
+    set_ignore_action_checks(True)
+    # proceed with test only when ros clock has been initialized
+    while rospy.get_time() == 0:
+      rospy.sleep(0.1)
+
```

Comment:
I liked how `test_task_scoop_circular.py` started with scooping above the terrain. Can we do that here too?

Reply:
Outside the scope of this PR, but the linear scoop looks much like a circular one to me, at least in that most of the dig seems to be complete in the circular and not linear movement.  I recall linear and circular looking more different in the past, but maybe it's the parameter values in these tests.

Reply:
> 

I can change the 'length' parameter to make it more different.

Reply:
Honestly I wouldn't bother. The thing is both of these tests will have to be at least slightly modified after I merge in the branch `OCEANWATER-1146_new_dig_model_integration`. This is because I rewrote the path planning for **TaskScoopLinear**, **TaskScoopCircular**, and **TaskGrind**  in that branch and as a result parameters will have to vary slightly from what we previously used. Thankfully I don't anticipate this to be a whole lot of work. 

It's worth noting that in the "new dig model" the shape of dig trajectories are invariant to their position in the workspace; however, as they are in `noetic-devel` you cannot claim that because there are some movements that command joint position instead of end-effector pose.